saturnbase-r-gpu-11.1/Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM rocker/cuda:4.1.2-cuda11.1
```

Comment:
This is GPLv2-licensed (https://github.com/rocker-org/ml/blob/master/LICENSE), but I _think_ it's ok - so long as we keep this repo public.